docs/BRP-LO GBA mapping - Personen gba.csv

```diff
@@ -0,0 +1,137 @@
+categorie;element;leveren door RvIG;Beperkt zoekresultaat;Persoon GBA variant;;;;;opmerking/issue;
+;;vaste waarde: GbaPersoon | GbaPersoonBeperkt;G;type;;;;;;
+1;01.10;ongewijzigd leveren;;aNummer;;;;;;
+1;01.20;ongewijzigd leveren;G;burgerservicenummer;;;;;;
+1;02;;;naam;;;;;;
```

Comment:
ik zou elementnummer weglaten bij groepen. het gaat om mapping elementnummer op een veld

Reply:
Gedaan, in beide csv bestanden.